WowCharacterComparer.Tests/Logic/User/PasswordValidationManagerTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using WowCharComparerWebApp.Configuration;
+using WowCharComparerWebApp.Logic.User;
+using Xunit;
+using System.Linq;
+
+namespace WowCharacterComparer.Test
+{
+    public class PasswordValidationManagerTest
+    {
+        [Theory]
+        [InlineData("Infermus123@vp.pl", true, "")]
+        [InlineData("Infermus123%vp.pl", false, UserMessages.UserEmailInvalidFormat)]
+        public void CheckUserEmail_WhenVariousUserInput_ShouldReturnExpectedBehaviour(string userEmail, bool status, string message)
```

Comment:
Not very creative to use almost the same method name in whole file. Please try to describe them properly.